libs/SalesforceReact/SalesforceReact.xcodeproj/project.pbxproj

```diff
@@ -313,6 +315,20 @@
 		};
 /* End PBXContainerItemProxy section */
 
+/* Begin PBXCopyFilesBuildPhase section */
```

Comment:
The `SalesforceReactTestApp` should embed the `SalesforceReact` framework binary like the other test targets.